pulley/src/interp.rs

```diff
@@ -493,6 +493,14 @@ impl XRegVal {
         self.0.u64 = x.to_le();
     }
 
+    pub fn set_f32(&mut self, x: f32) {
+        self.0.u32 = x.to_bits().to_le();
+    }
+
+    pub fn set_f64(&mut self, x: f64) {
+        self.0.u64 = x.to_bits().to_le();
+    }
+
```

Comment:
Mind also deleting these setters too? (fine to do in a follow-up too)

Reply:
Done.